src/mongodb/validator.js

```diff
@@ -0,0 +1,17 @@
+import * as joi from 'joi';
+
+export default dbUrl => {
+  const { error, value } = joi.validate(
```

Comment:
This can be replaced with `joi.attempt`. The error object can be passed as the third argument.